### DISCUSSION TITLE
Is the website and project safe?

### DISCUSSION THREAD

**Discussion by mshelto59** [nuxsmin/sysPass#1802] [Q&A] [answered]
I noticed that the browser tab for the sysPass.org website reads 
"sysPass| Hacked by tokeichun"
Do we know if the project is currently safe to use?
I'm assuming this has not affected the github repo, but I wanted to make the developers aware of this.

![image](https://user-images.githubusercontent.com/14967457/151712520-62611191-c7b5-419a-a399-7be8fd8a8bbd.png)

**Comment by mshelto59** [comment, marked as answer]
This appears to have disappeared and is no longer present on the website title bar.

**Comment by nuxsmin** [comment]
Hello, thanks for the notice!!

It seems that someone wanted to play with sysPass website. I have checked all the site files and updated Grav to the latest version 1.7.29 (it was on v1.6.30 which was one step down from v1.6.31).

sysPass demo and binaries (on GitHub) are not affected.
